### PR TITLE
Delay docs

### PR DESCRIPTION
**Pull request by Armenvardanyan95**
A minor doc change proposed by @BioPhoton in #4921 

(Also automatic linter fixes from the post-commit hook)

Related issue: https://github.com/ReactiveX/rxjs/issues/4921

### CODE REVIEW

File: src/internal/operators/delay.ts [hunk 1]
```diff
@@ -41,8 +41,15 @@ import { timer } from '../observable/timer';
  * delayedClicks.subscribe(x => console.log(x));
  * ```
  *
- * @see {@link debounceTime}
  * @see {@link delayWhen}
+ * @see {@link throttle}
```
Comment (niklas-wortmann) [inline src/internal/operators/delay.ts:45]: there are a lot of formatation changes  in this pr that are entirely unrelated to this, I'd appreciate it if you could revert those

Comment (Armenvardanyan95) [reply src/internal/operators/delay.ts:45]: The post-commit hook does that for me automatically, I did not want to interfere. Is there a way I can disable it? I ill revert those

Comment (Armenvardanyan95) [reply src/internal/operators/delay.ts:45]: It got a bit messy, sorry for that. I will submit another PR shortly

Comment (Armenvardanyan95) [reply src/internal/operators/delay.ts:45]: Here is the new PR, containing only the documentation change: https://github.com/ReactiveX/rxjs/pull/6156
